tests/test_vlan.py

```diff
@@ -280,91 +194,88 @@ def test_AddVlanWithIncorrectKeyPrefix(self, dvs, test_input, expected):
             # TODO: refactor to use loganalyzer
             self.check_syslog(dvs, marker, "vlanmgrd",
                               "Invalid key format. Not a number after \'Vlan\' prefix:", vlan, 1)
         else:
-            self.remove_vlan(vlan_id)
-            self.get_and_verify_vlan_ids(0)
+            self.dvs_vlan.remove_vlan(vlan_id)
+            self.dvs_vlan.get_and_verify_vlan_ids(0)
 
     def test_AddPortChannelToVlan(self, dvs):
-        self.setup_db(dvs)
 
         vlan = "2"
         lag_member = "Ethernet0"
         lag_id = "0001"
         lag_interface = "PortChannel{}".format(lag_id)
 
-        self.create_port_channel(lag_id)
-        lag_entries = self.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_LAG", 1)
+        self.dvs_lag.create_port_channel(lag_id)
+        lag_entries = self.dvs_vlan.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_LAG", 1)
 
-        self.create_port_channel_member(lag_id, lag_member)
+        self.dvs_lag.create_port_channel_member(lag_id, lag_member)
 
         # Verify the LAG has been initialized properly
-        lag_member_entries = self.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER", 1)
-        fvs = self.asic_db.wait_for_entry("ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER", lag_member_entries[0])
+        lag_member_entries = self.dvs_vlan.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER", 1)
+        fvs = self.dvs_vlan.asic_db.wait_for_entry("ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER", lag_member_entries[0])
         assert len(fvs) == 4
         assert fvs.get("SAI_LAG_MEMBER_ATTR_LAG_ID") == lag_entries[0]
-        assert self.asic_db.port_to_id_map[fvs.get("SAI_LAG_MEMBER_ATTR_PORT_ID")] == lag_member
+        assert self.dvs_vlan.asic_db.port_to_id_map[fvs.get("SAI_LAG_MEMBER_ATTR_PORT_ID")] == lag_member
 
-        self.create_vlan(vlan)
-        self.get_and_verify_vlan_ids(1)
+        self.dvs_vlan.create_vlan(vlan)
+        self.dvs_vlan.get_and_verify_vlan_ids(1)
 
-        self.create_vlan_member(vlan, lag_interface, "tagged")
-        self.get_and_verify_vlan_member_ids(1)
+        self.dvs_vlan.create_vlan_member(vlan, lag_interface, "tagged")
+        self.dvs_vlan.get_and_verify_vlan_member_ids(1)
 
-        self.remove_vlan_member(vlan, lag_interface)
-        self.get_and_verify_vlan_member_ids(0)
+        self.dvs_vlan.remove_vlan_member(vlan, lag_interface)
+        self.dvs_vlan.get_and_verify_vlan_member_ids(0)
 
-        self.remove_vlan(vlan)
-        self.get_and_verify_vlan_ids(0)
+        self.dvs_vlan.remove_vlan(vlan)
+        self.dvs_vlan.get_and_verify_vlan_ids(0)
 
-        self.remove_port_channel_member(lag_id, lag_member)
-        self.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER", 0)
+        self.dvs_lag.remove_port_channel_member(lag_id, lag_member)
+        self.dvs_vlan.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER", 0)
```

Comment:
This is a little odd, but I'm planning on refactoring more of the LAG stuff in the next couple weeks so this is OK. (This is just a mental note to go back and fix this when I'm done. 🚀)